ui/cypress/e2e/variables.test.ts

```diff
@@ -201,6 +201,22 @@ describe('Variables', () => {
     cy.getByTestID(`variable-card--name ${queryVariableName}`).contains(
       queryVariableName
     )
+
+    //create variable by uploader
+    cy.getByTestID('add-resource-dropdown--button').click()
+
+    cy.getByTestID('add-resource-dropdown--import').click()
+
+    const yourFixturePath = 'data-for-variable.json'
+    cy.get('.drag-and-drop').attachFile(yourFixturePath, {
```

Comment:
Out of curiosity, is the drag and drop available to add a test ID to? Just wanna make sure we follow best practices if it is. If it’s possible to add a testID and select it based on that then that’d be great, otherwise this is gold

Reply:
I did try to do it by test id, but either the test id for the drag and drop was wrong or the library was not compatible as it did not seem to work. The library does not have anything about it, so im not sure who is to blame. When i use it to test somewhere else ill try it again and see how it goes